src/js/components/Form/index.d.ts

```diff
@@ -16,6 +16,7 @@ export interface FormProps<T> {
   onValidate?: (validationResults: {
     errors: Record<string, any>;
     infos: Record<string, any>;
+    isFormValid: boolean;
```

Comment:
I think we should name this just "valid". We are already in a Form context, so we don't need that term. And, we tend to just use the adjective alone without the "is" prefix elsewhere, such as
```
<Button>
  {({ disabled, hover, focus }) => {}}
</Button>
```

Reply:
Renaming isFormValid to valid